pkg/config/config.go

```diff
@@ -878,9 +876,7 @@ func GetResources(component string, advanced *observabilityv1beta2.AdvancedConfi
 
 func GetOBAResources(oba *mcoshared.ObservabilityAddonSpec) *corev1.ResourceRequirements {
 	cpuRequests := MetricsCollectorCPURequets
-	cpuLimits := MetricsCollectorCPULimits
```

Comment:
How about just set cpuLimits and memoryLimits to be 0? it should be working w/o changes. correct?